packages/plugin-apex/src/legacyColorization.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as chalk from 'chalk';
+import { Logger } from '@salesforce/core';
+
+const DEFAULT_COLOR_MAP = {
+  CONSTRUCTOR_: 'magenta',
+  EXCEPTION_: 'red',
+  FATAL_: 'red',
+  METHOD_: 'blue',
+  SOQL_: 'yellow',
+  USER_: 'green',
+  VARIABLE_: 'cyan'
+};
+
+/**
+ * @description this is a holdover from the toolbelt API, which allows for custom colorization of the logs.
+ * @param log - full debug log retrieved from an org.
+ * @returns colorized log
+ */
+export async function colorizeLog(log: string): Promise<string> {
+  const logger = await Logger.child('apexLogApi', { tag: 'tail' });
+  // default color registry
```

Comment:
i think we can remove this comment, the variable names are descriptive

Reply:
same with the others in this method, we generally try to reserve comments to provide _context_ - [this article](https://blog.codinghorror.com/coding-without-comments/) explains it better

Reply:
Totally agree! A lot of this was copy/paste; I'm hesitant to refactor it the way that it deserves since hopefully we'll be able to switch this to use the better colorization that we use elsewhere in the library. It's also not well tested, and @smaddox-sf said not to waste time testing it with this move. But I am happy to strip the comments for now 👍